tests/v1/kv_connector/unit/test_lmcache_connector.py

```diff
@@ -0,0 +1,756 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+from unittest.mock import MagicMock
+
+import pytest
+
+from vllm.distributed.kv_events import BlockStored
+from vllm.distributed.kv_transfer.kv_connector.v1.lmcache_connector import (
+    LMCacheConnectorV1,
+    LMCacheKVEvents,
+)
```

Comment:
qq: is the lmcache package actually needed to run these tests or is it just for importing the types to insert in the mocks?

Reply:
Good catch, thanks @NickLucche. Updated to do a lazy import in the connector and therefore not requiring lmcache package during the mocking.